assets/scss/components/citadel/forms/_forms.scss

```diff
@@ -246,7 +246,6 @@
     }
 
     .form-prefixPostfix-button--postfix {
```

Comment:
I think it's removed by accident

Reply:
@bc-alexsaiannyi there was an extra space and stylelint (or prettier) automatically removed this indentation )